litestar/testing/client/async_client.py

```diff
@@ -30,8 +31,11 @@
 
 T = TypeVar("T", bound=ASGIApp)
 
+# ensure that httpx logging is not interfering with our test client
+logging.getLogger("httpx").setLevel(logging.WARNING)
```

Comment:
This is going to break all httpx' logging for all users once they import this module.

It should go into our tests, not the test clients.